data/filesystem/autoyast_filesystem_withouthome.xml.ep

```diff
@@ -6,7 +6,7 @@
     <email/>
     <reg_code>{{SCC_REGCODE}}</reg_code>
     <install_updates config:type="boolean">true</install_updates>
-    % unless ($check_var->('FLAVOR', '') =~ /-Updates$|-Incidents/) {
+    % unless ($get_var->('FLAVOR') =~ /-Updates$|-Incidents/) {
```

Comment:
Actually it's still good to use `get_var($variable, $default)` so here it would be `$get_var->('FLAVOR', '')` to prevent a warning about comparing undef by regex in case flavor is not defined.